includes/deprecated/deprecated-functions.php

```diff
@@ -256,3 +256,32 @@ function give_get_purchase_form_user( $valid_data = array() ) {
 	give_get_donation_form_user( $valid_data );
 
 }
+
+/**
+ * Give Checkout Button.
+ *
+ * Renders the button on the Checkout.
+ *
+ * @since  1.0
+ * @deprecated 1.8.8
+ *
+ * @param  int $form_id The form ID.
+ *
+ * @return string
+ */
+function give_checkout_button_purchase( $form_id ) {
+	$backtrace = debug_backtrace();
+
+	_give_deprecated_function( __FUNCTION__, '1.8.8', null, $backtrace );
+
+	$display_label_field = give_get_meta( $form_id, '_give_checkout_label', true );
```

Comment:
@ravinderk why don't you just call the renamed function `give_get_donation_form_submit_button()` within here without repeating the code?